tasks/create_storage_domain.yml

```diff
@@ -1,6 +1,11 @@
 ---
 - name: Create hosted engine local vm
   block:
+  - name: Wait until all interfaces wake up
+    wait_for:
+      timeout: 150
+    delegate_to: localhost
```

Comment:
delegating to localhost won't work when deploying from a remote host(using Ansible role), you need to change this to he_ansible_host_name.